cmd/monaco/integrationtest/v2/all_configs_integration_test.go

```diff
@@ -38,6 +38,7 @@ func TestIntegrationAllConfigsClassic(t *testing.T) {
 	manifest := configFolder + "manifest.yaml"
 
 	t.Setenv(featureflags.OpenPipeline.EnvName(), "true")
+	t.Setenv(featureflags.ServiceLevelObjective.EnvName(), "true")
```

Comment:
I don't think this will do anything - same with the OpenPipeline a line above. 

Without looking deeper, I would think that `TestIntegrationAllConfigs**Classic**` does not use OAuth credentials, and thus the API is not available with API Tokens. 

Thus, I don't think those two lines are actually necessary

Reply:
Actually it's still needed. Here all conifgs are read, including that ones that are not really needed and an invalid type would result in an error